ToDoApp/ToDoApp/ToDoService.swift

```diff
@@ -0,0 +1,54 @@
+//
+//  ToDoDetail.swift
+//  ToDoApp
+//
+//  Created by Trong Hieu Vuong on 2022-11-29.
+//
+
+import Foundation
+
+enum TaskError: Error {
+    case unavailableTask
+}
+
+struct Task {
+    var id: String = ""
+    var name: String = ""
+    var deadline: Date = Date.now
+    var status: Bool = false
+    init(name: String, deadline: Date?) {
+        self.name = name
+        self.deadline = deadline!
```

Comment:
there should be a good reason to use force unwrap, currently I don't see it, since nil may passed to the initializer and the app will crash

Reply:
This has been fixed.